photon_installer/isoInstaller.py

```diff
@@ -54,7 +59,13 @@ def __init__(self, options):
                 return
 
         if ks_path:
-            install_config=self._load_ks_config(ks_path)
+            install_config = self._load_ks_config(ks_path)
+
+        # insecure_installation flag added through commandline overrides that of ks_config
+        if k_param:
```

Comment:
k_param not required. We can use self.insecure_installation here

Reply:
No here it is required to know whether insecure_installation parameter comes from command line if not we dont edit install_config. But better way will be to use insecure_installation=None and see if it exits(passed in command line). Will edit

Reply:
Done